lib/ReportDataCollector.php

```diff
@@ -130,20 +139,36 @@ public function getReportJson($options = JSON_PRETTY_PRINT, $depth = 512) {
 	}
 
 	/**
+	 * @param array $report
+	 */
+	public function addEventListenerReportData(&$report) {
+		foreach ($this->appConfigData as $index) {
+			foreach ($index as $innerKey => $innerVal) {
+				$report[$innerKey] = $innerVal;
```

Comment:
This allows to overwrite existing report keys, your if from before was okay, just needs to be negated. Add it if the key does not exist yet.